### ISSUE TITLE
[A11y] The headings of the news cards should be in H3 instead of H4 to avoid skipping heading hierarchy

### ISSUE THREAD

**Issue by michaelktiufb** [hlxsites/accenture-newsroom#422]
1. Navigate to https://tempnewsroom.accenture.com/
2. Check the news card headlines

ER: Headlines are in H3
AR: Headlines were in H4

**Comment by amol-anand** [comment]
current production newsroom.accenture.com is also using H4 for the news card headlines. same in our site.

![Screenshot 2023-11-03 at 5 23 34 PM](https://github.com/hlxsites/accenture-newsroom/assets/7074782/19d62f91-a79a-4e26-aba1-8d427f7dd3e5)
